charts/et-hearings-api/values.preview.template.yaml

```diff
@@ -2,3 +2,11 @@ java:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  keyVaults:
+  #  et-hearings-api:
+  #    secrets:
+  #      - name: et-hearings-api-s2s-secret
+  #        alias: ET_HEARINGS_S2S_SECRET
```

Comment:
Is this needed at all if its declared in `values.yml` too?

Reply:
removed those commented out lines